resources/models/resource.py

```diff
@@ -203,6 +203,8 @@ class Resource(ModifiableModel, AutoIdentifiedModel):
                                         default=ACCESS_CODE_TYPE_NONE)
     reservable_days_in_advance = models.PositiveSmallIntegerField(verbose_name=_('Reservable days in advance'),
                                                                   null=True, blank=True)
+    reservable_min_days_in_advance = models.PositiveSmallIntegerField(
+        verbose_name=_('Days until reservation is possible'), null=True, blank=True)
```

Comment:
I'm interpreting "days until reservation is possible" to mean "how many days you have to wait until you can make a reservation". This interpretation would be sad indeed, because no matter how many days the poor user waits, the promised land of resource reservation will always be `reservable_min_days_in_advance` days in the future.

I suggest we rename `reservable_days_in_advance` to `reservable_max_days_in_advance` while retaining `reservable_days_in_advance` as a legacy API field. The verbose names would then be "Reservable max. days in advance" and "Reservable min. days in advance".

Reply:
Verbose names changed in 4ba3c7.

@juyrjola What is the way of working in this project regarding commit squashing? Is the developer expected to rewrite the history of the feature branch when it's deemed done, or should the merger do a rebase+squash?

Reply:
In the future it's also possible to work in a fork, squash there and push to the upstream repo only after the feature is ready for review, if few commits are preferred.

Reply:
This was discussed with @jussiarpalahti and the preferred way is to squash-merge to master.